content/articles/overview-of-a-react-application.md

```diff
@@ -53,7 +53,7 @@ date with its state and props. `props` refers to the parameters passed
 to the component by its parent on initialization. A component doesn't
 change its own props, it just uses whatever props its parent gave it.
 The component can change its own `state`, though, using
-`this.setState()`. `state` is how you model changes over time in React
+`this.setState()`. `state` is how your model changes over time in React
```

Comment:
actually.. i was using model as a verb here, so it wasn't a typo. maybe i should rephrase it if it looked like a typo, though.